requirements.txt

```diff
@@ -0,0 +1,45 @@
+attrs==21.2.0
```

Comment:
You've added a `requirements.txt` file which is usually used for Pip to manage projects, but we're tracking our dependencies in Poetry; can we get rid of this?